index.js

```diff
@@ -137,4 +137,38 @@ newSightingForm.addEventListener('submit', function (event) {
 /***** End of Starter Code *****/
 /************************** EVENTS PART 2 JS MINI CHALLENGE ******************************/
 
+// Deliverable 1 + 2 : Like an animal sighting + Delete an animal sighting
+// When a user clicks an animal sighting's like button, that animal sighting's likes should increase by 1.
+// When a user clicks an animal sighting's delete button, that animal sighting should be removed from the page. 
+
+
+const animalsUl = document.querySelector('ul#animals')
+
+animalsUl.addEventListener('click', function(event) {
+
+    if (event.target.matches('button.like-button')) {
+        const pLikesDisplay = event.target.previousElementSibling 
+        const likes = parseInt(pLikesDisplay.textContent) + 1
+        pLikesDisplay.textContent = `${likes} Likes`
+    } 
+    else if (event.target.matches('button.delete-button')) {
+        const li = event.target.closest('li')
+        li.remove()
+    } 
+    else if (event.target.matches('button.toggle-update-form-button')) {
+        const updateForm = event.target.nextElementSibling
+        if (updateForm.style.display === 'none') {
+            updateForm.style.display = 'block'
+        } else {
+            updateForm.style.display = 'none'
+        }
+    }
+})
+
 
+//BONUS: Update animal sighting description
+
+animalsUl.addEventListener('submit', function(event) {
+    event.preventDefault()
+    event.target.closest('li').querySelector('p').textContent = event.target.querySelector('input').value
+})
```

Comment:
I would be careful about trying to do too many things on 1 line. It can make it more difficult for someone else to read your code. Great work here as well. You are listening for any form submission that occurs within the animals UL element. Then you make sure to prevent the default action of the form. Then you find the p tag and update it's value based on the form data. Don't forget to change the form field value to the new description. Again, we can break it apart into 2 steps.

```
/***** BONUS DELIVERABLE part 1 *****/

animalsUL.addEventListener("submit", function(event){
    event.preventDefault()
    if (event.target.matches('.update-form')){
        handleUpdateFormSubmission(event)
    }
})

/***** BONUS DELIVERABLE part 2 *****/
function handleUpdateFormSubmission(e){
    e.preventDefault()
    const description = e.target.children[0].value
    const animalDescriptionPTag = e.target.parentElement.children[0]
    animalDescriptionPTag.innerText = description
    e.target.value = description
}
```